libtcmu.c

```diff
@@ -277,78 +268,43 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	case MODE_SELECT:
 	case MODE_SELECT_10:
 		return tcmu_emulate_mode_select(cdb, iovec, iov_cnt, sense);
+	default:
+		tcmu_err("unknown command %x\n", cdb[0]);
+		return TCMU_NOT_HANDLED;
+	}
+}
+
+static bool command_is_store_call(uint8_t cmd)
+{
```

Comment:
I don't think we need a store call out abstraction if we come along with pass through mechanism. If rbd driver want to implement it on its own over the generic one, we just need to pass down it to rbd.c without affecting other handlers at all. With the call out abstraction, at least we need to call extra function in every handlers, which is not necessary for now.